.github/workflows/build.yaml

```diff
@@ -1,13 +1,18 @@
 name: Build binaries
 
 on:
+  pull_request_target:
```

Comment:
Isn't this a security hole? E.g., a malicious user could write a code that steals our secrets and send a PR.

Reply:
dependabot secrets are stored separate from the default ones: 

![image](https://user-images.githubusercontent.com/3393533/192732497-7a11aaee-f610-4bad-be4d-9bffa9dc1cd6.png)

but any PR (from external users) would still need to be approved by repo owner before the CI picks up the job. The same is true for `pull_request` based trigger.

Reply:
Ok, then it makes sense.